internal/pkg/cli/task_run.go

```diff
@@ -256,8 +306,14 @@ func (o *runTaskOpts) Execute() error {
 }
 
 func (o *runTaskOpts) runTask() ([]string, error) {
-	// TODO: run tasks
-	return nil, nil
+	o.spinner.Start(fmt.Sprintf("Waiting for %s to be running.", o.groupName))
```

Comment:
What do you think of:
```suggestion
	o.spinner.Start(fmt.Sprintf("Waiting for %d tasks to be running for %s.", o.count, o.groupName)) // whatever is the flag for num of tasks
```